Buildings/Media/Examples/BaseClasses/PartialProperties.mo

```diff
@@ -11,6 +11,7 @@ partial model PartialProperties
   parameter Modelica.SIunits.Pressure p = Medium.p_default "Pressure";
   parameter Modelica.SIunits.MassFraction X[Medium.nX]=
     Medium.X_default "Mass fraction";
+  parameter Real errAbs=1E-8 "Absolute error used in the check of the state calculations";
```

Comment:
The format of this model seems strange to me.

Reply:
I agree at times; however, this was a model that existed in the MBL previously, so I did not change the formatting.